shared/router-v2/header/index.desktop.js

```diff
@@ -83,9 +83,12 @@ const styles = Styles.styleSheetCreate({
       alignItems: 'center',
     },
   }),
-  icon: {
-    marginRight: 6,
-  },
+  icon: Styles.platformStyles({
+    isElectron: {
+      ...Styles.desktopStyles.windowDraggingClickable,
```

Comment:
i don't think we want the entire header to be draggable (and therefore we need to add this style to all clickable things in the header). i think we just want the top part (where the chrome used to be) to be draggable